source/acquaman/AMGenericScanActionControllerAssembler.cpp

```diff
@@ -156,7 +168,13 @@ AMAction3* AMGenericScanActionControllerAssembler::generateActionTreeForStepAxis
 
 	if (axisControl){
 
-		AMAction3 *regionStart = AMActionSupport::buildControlMoveAction(axisControl, stepScanAxisRegion->regionStart());
+		AMAction3 *regionStart = 0;
+
+		if (direction_ == AMScanConfiguration::Increase)
```

Comment:
I wonder, is it possible for the `direction_` to be out of sync with the regions? I mean, could somebody manually enter regions that go backwards and `direction_ == AMScanConfiguration::Increase`?